client/src/panels/panel_declarations/finances/budget_measures/budget_measures_panel.js

```diff
@@ -876,18 +876,16 @@ class BudgetMeasureHBars extends React.Component {
             </div>
           </div>
           <MediaQuery minWidth={992}>
-            <div className="centerer" style={{height: `${data.length*30 + 150}px`}}>
```

Comment:
Hmm, so I noticed `centerer` class is gone, that doesn't affect anything?

Reply:
nope